### ISSUE TITLE
Vibration

### ISSUE THREAD

**Issue by rasppliances** [Open-Smartwatch/open-smartwatch-os#50]
Hello,

Has anyone looked into adding vibration? I know any motor would add size to what is already a good sized watched, but if you don't have vibration, getting notifications on your watch won't help.

I think that a small transistor could be added to the circuit board, hooked up to an open GPIO (assuming there is one), and used to control vibration through PWM.  We could add an osw_vibration library to be included in apps.  I imagine the call would be really simple, like doVibration(intensity, duration);  Maybe it could even go into osw_ui_util.cpp (I know it's not a graphical UI element, but it is a UI element since it's interacting with the user:) ).

Rick

**Comment by uvwxy** [comment]
Yes, I'm looking into adding a vibration motor to the GPS edition. The PCB will be bigger, so there is room for one.
I'll have to give up the second interrupt pin from the accelerometer to do this though :).
